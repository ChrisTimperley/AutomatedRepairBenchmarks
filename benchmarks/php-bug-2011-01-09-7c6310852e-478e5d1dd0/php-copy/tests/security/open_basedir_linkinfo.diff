006+ 
007+ Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 10
008+ bool(false)
009+ 
010+ Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 14
011+ int(-1)
012+ 
013+ Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 15
014+ int(-1)
015+ 
016+ Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 16
017+ int(-1)
018+ 
019+ Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 17
020+ int(-1)
021+ 
022+ Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 18
023+ int(-1)
024+ 
025+ Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 22
026+ bool(false)
027+ 
028+ Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 23
029+ int(-1)
030+ 
031+ Warning: unlink(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/ok/symlink.txt): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 24
006- bool(true)
007- int(%d)
008- int(%d)
009- int(%d)
010- int(%d)
011- int(%d)
012- 
013- Warning: symlink(): open_basedir restriction in effect. File(%s/test/bad/bad.txt) is not within the allowed path(s): (.) in %s on line %d
015- int(%d)
016- 
017- Warning: unlink(): open_basedir restriction in effect. File(%s/test/ok/symlink.txt) is not within the allowed path(s): (.) in %s on line %d
018- bool(false)