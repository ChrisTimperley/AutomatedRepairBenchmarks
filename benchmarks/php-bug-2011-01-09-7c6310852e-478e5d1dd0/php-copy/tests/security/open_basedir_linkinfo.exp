*** Testing open_basedir configuration [linkinfo] ***
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
int(%d)
int(%d)
int(%d)
int(%d)
int(%d)

Warning: symlink(): open_basedir restriction in effect. File(%s/test/bad/bad.txt) is not within the allowed path(s): (.) in %s on line %d
bool(false)
int(%d)

Warning: unlink(): open_basedir restriction in effect. File(%s/test/ok/symlink.txt) is not within the allowed path(s): (.) in %s on line %d
bool(false)
*** Finished testing open_basedir configuration [linkinfo] ***