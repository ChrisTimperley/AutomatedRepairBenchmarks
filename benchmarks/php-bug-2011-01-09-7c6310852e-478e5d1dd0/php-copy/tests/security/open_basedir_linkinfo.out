*** Testing open_basedir configuration [linkinfo] ***
bool(true)
bool(true)
bool(true)
bool(true)

Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 10
bool(false)

Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 14
int(-1)

Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 15
int(-1)

Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 16
int(-1)

Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 17
int(-1)

Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 18
int(-1)

Warning: symlink(): Protocol error in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 22
bool(false)

Warning: linkinfo(): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 23
int(-1)

Warning: unlink(/vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/test/ok/symlink.txt): No such file or directory in /vagrant/benchmarks/php-bug-2011-01-09-7c6310852e-478e5d1dd0/php/tests/security/open_basedir_linkinfo.php on line 24
bool(false)
*** Finished testing open_basedir configuration [linkinfo] ***